diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2f9a21e4f..4f2fba85c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -28,6 +28,7 @@
 if(CMAKE_SYSTEM_PROCESSOR)
 	if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "^arm")
 		set(ARM ON)
+		add_definitions(-mfpu=neon -fomit-frame-pointer -ftree-vectorize -mvectorize-with-neon-quad -ffast-math -DARM_NEON)
 			if(UNIX AND NOT APPLE)
 				execute_process(COMMAND cat /proc/cpuinfo OUTPUT_VARIABLE OUTSTR)
 				string(FIND "${OUTSTR}" "ODROID-XU" pos)
