--- a/CMakeLists.orig.txt	2020-06-27 19:48:53.570256414 +0200
+++ b/CMakeLists.txt	2020-06-27 21:39:47.892536425 +0200
@@ -272,7 +272,6 @@
 	# Disable some warnings
 	add_definitions(-Wno-multichar)
 	add_definitions(-Wno-deprecated-register)
-	add_definitions(-Wno-tautological-pointer-compare)
 
 	# Don't compile with strict aliasing, we're not 100% aliasing-safe
 	add_compile_options(-fno-strict-aliasing)
